target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-gl-ap1300.dts

```diff
@@ -14,7 +14,6 @@
 		led-failsafe = &led_power;
 		led-running = &led_power;
 		led-upgrade = &led_power;
-		label-mac-device = &gmac0;
```

Comment:
Which MAC does now feature the label-MAC?

Reply:
@blocktrron @PolynomialDivision 
The label MAC shows up as the MAC of eth0, which is correct.
wan, lan and br-lan all share this mac address as would be expected.
The two radios have incremental MAC addresses based on the label MAC.
The 4g/lte modem has its own MAC.
Probably the easiest way to show this is the output if `ip addr`:

```
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP qlen 1000
    link/ether 94:83:c4:04:b5:1a brd ff:ff:ff:ff:ff:ff
    inet6 fe80::9683:c4ff:fe04:b51a/64 scope link 
       valid_lft forever preferred_lft forever
3: lan@eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master br-lan state UP qlen 1000
    link/ether 94:83:c4:04:b5:1a brd ff:ff:ff:ff:ff:ff
4: wan@eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP qlen 1000
    link/ether 94:83:c4:04:b5:1a brd ff:ff:ff:ff:ff:ff
    inet 192.168.8.198/24 brd 192.168.8.255 scope global wan
       valid_lft forever preferred_lft forever
    inet6 fe80::9683:c4ff:fe04:b51a/64 scope link 
       valid_lft forever preferred_lft forever
5: wwan0: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN qlen 1000
    link/ether 36:6d:17:24:cd:e7 brd ff:ff:ff:ff:ff:ff
8: br-lan: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP qlen 1000
    link/ether 94:83:c4:04:b5:1a brd ff:ff:ff:ff:ff:ff
    inet 192.168.1.1/24 brd 192.168.1.255 scope global br-lan
       valid_lft forever preferred_lft forever
    inet6 fdaf:b5ee:6e9f::1/60 scope global noprefixroute 
       valid_lft forever preferred_lft forever
    inet6 fe80::9683:c4ff:fe04:b51a/64 scope link 
       valid_lft forever preferred_lft forever
9: wlan1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master br-lan state UP qlen 1000
    link/ether 94:83:c4:04:b5:1d brd ff:ff:ff:ff:ff:ff
    inet6 fe80::9683:c4ff:fe04:b51d/64 scope link 
       valid_lft forever preferred_lft forever
10: wlan0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master br-lan state UP qlen 1000
    link/ether 94:83:c4:04:b5:1c brd ff:ff:ff:ff:ff:ff
    inet6 fe80::9683:c4ff:fe04:b51c/64 scope link 
       valid_lft forever preferred_lft forever

```

Reply:
@blocktrron I thought this is not anymore necessary because I provide the nvme cells?